snake-odin/src/game.odin

```diff
@@ -0,0 +1,71 @@
+package main
+
+import "w4"
+import "core:math/rand"
+import "core:runtime"
+
+fruit_sprite := [16]u8{ 0x00,0xa0,0x02,0x00,0x0e,0xf0,0x36,0x5c,0xd6,0x57,0xd5,0x57,0x35,0x5c,0x0f,0xf0 }
```

Comment:
Not necessarily needed but you could do this as it seems that the `fruit.img` is included
```
fruit_sprite := #load("fruit.img")
```

Reply:
The existence of the fruit.img was my mistake. I'll remove it. At first I tested it with the load-macro. But I didn't want it to be too far away from the other sources.